app/src/main/java/rs/ruffle/PanicScreen.kt

```diff
@@ -0,0 +1,77 @@
+package rs.ruffle
+
+import android.content.res.Configuration
+import androidx.compose.foundation.horizontalScroll
+import androidx.compose.foundation.layout.Arrangement
+import androidx.compose.foundation.layout.Column
+import androidx.compose.foundation.layout.fillMaxSize
+import androidx.compose.foundation.layout.fillMaxWidth
+import androidx.compose.foundation.layout.padding
+import androidx.compose.foundation.layout.wrapContentSize
+import androidx.compose.foundation.rememberScrollState
+import androidx.compose.foundation.text.selection.SelectionContainer
+import androidx.compose.foundation.verticalScroll
+import androidx.compose.material3.MaterialTheme
+import androidx.compose.material3.Scaffold
+import androidx.compose.material3.Text
+import androidx.compose.runtime.Composable
+import androidx.compose.ui.Alignment
+import androidx.compose.ui.Modifier
+import androidx.compose.ui.tooling.preview.Preview
+import androidx.compose.ui.unit.dp
+import rs.ruffle.ui.theme.RuffleTheme
+
+@Composable
+fun PanicScreen(message: String) {
+    Scaffold { innerPadding ->
+        Column(
+            modifier = Modifier
+                .padding(innerPadding)
+                .fillMaxSize(),
+            verticalArrangement = Arrangement.Center
+        ) {
+            Text(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .wrapContentSize(align = Alignment.Center),
+                style = MaterialTheme.typography.headlineLarge,
+                text = "Ruffle Panicked :("
+            )
+            SelectionContainer {
+                Text(
+                    modifier = Modifier
+                        .wrapContentSize(align = Alignment.Center)
+                        .padding(horizontal = 8.dp, vertical = 20.dp)
+                        .verticalScroll(rememberScrollState())
+                        .horizontalScroll(rememberScrollState()),
+                    text = message,
+                    softWrap = false
+                )
+            }
+        }
+    }
+}
+
+@Preview(name = "Panic - Light", uiMode = Configuration.UI_MODE_NIGHT_NO)
+@Preview(name = "Panic - Dark", uiMode = Configuration.UI_MODE_NIGHT_YES)
+@Composable
+fun PanicScreenPreview() {
+    RuffleTheme {
+        PanicScreen(
+            message = """Error: panicked at core/src/display_object/movie_clip.rs:477:9:
+assertion `left == right` failed: Called replace_movie on a clip with LoaderInfo set
+  left: Some(LoaderInfoObject(LoaderInfoObject { ptr: 0x31b30a8 }))
+ right: None
+    at n.wbg.__wbg_new_796382978dfd4fb0 (https://unpkg.com/@ruffle-rs/ruffle/core.ruffle.90db0a0ab193ed0c601b.js:1:83857)
```

Comment:
_A JS file?! In my native application?! :scream:_

Reply:
I copied a random stacktrace from a recent ruffle issue for somewhat realistic data to visualize 😅

Reply:
I know... ;) :D